ironfish-cli/src/multisigBroker/clients/client.ts

```diff
@@ -178,23 +178,37 @@ export abstract class MultisigClient {
     maxSigners: number,
     minSigners: number,
     identity: string,
+    allowedIdentities?: string[],
```

Comment:
It feels a _little_ strange that the DKG functions take these params even though they are technically unsupported. I'm not sure if I think that's better, or if the DKG flow functions should not take the param at all.

Reply:
I think the only reason to have `allowedIdentities` in DKG is to try to prevent one person from joining the session more than once. 🤔 

In practice I think it's going to be difficult to provide a good experience for setting the list of allowed identities for DKG, so I'm going to remove it from the DKG methods for now